src/runmodes.c

```diff
@@ -147,6 +147,12 @@ static const char *RunModeTranslateModeToName(int runmode)
 #endif
         case RUNMODE_UNIX_SOCKET:
             return "UNIX_SOCKET";
+        case RUNMODE_WINDIVERT:
+#ifdef WINDIVERT
+            return "WINDIVERT";
```

Comment:
indent looks off

Reply:
I'm not seeing it?
![image](https://user-images.githubusercontent.com/3700215/37416198-63b5c014-276a-11e8-9463-56d1579a33cc.png)

Reply:
Yeah you are right, please disregard.